sox.features.info.json

```diff
@@ -170,77 +171,77 @@
             "desc": "Open imgur images as larger modals on click",
             "extended_description": "When clicking on an image that is hosted on Stack Exchange's imgur account, a modal will open showing a larger version of the image. If you still want to open the image in a new tab, there is a 'source' link in the modal.",
             "meta": "",
-            "match": "*://*/questions*,*://*/review/*",
-            "exclude": ""
+            "match": "*://*/questions/*,*://*/review/*",
+            "exclude": "*://*/questions*tab=*,*://chat.*.com/*"
```

Comment:
I'm not sure the `*://*/questions*tab=*` parts are necessary for all the features in which you've already added a trailing `/` to the `*://*/questions/*` for the `match` property, do you think we can get rid of this?

Reply:
I'm also not sure we actually need the `*://chat.*.com/*` exclusions for all the features in which we've already got a specific `match` for a question page -- the `exclude` is mainly for cases where the `match` matches, but we want to 'override' and exclude specific pages under the matched part!

Do you think we can remove these too?

Reply:
Good catches! Working on them ;)